editors/vscode/src/common.ts

```diff
@@ -130,6 +123,18 @@ export function languageClientOptions(
     };
 }
 
+export function use_wasm_preview(): boolean {
+    if (process.env.hasOwnProperty("CODESPACES")) {
+        return true;
+    }
+
+    const result = vscode.workspace
+        .getConfiguration("slint")
+        .get("preview.providedByEditor", true);
```

Comment:
You're changing the default to true? Is that intentional?  But this is not defaulted to true in the package.json afaik, so this is inconsistant.